specification/automanage/resource-manager/Microsoft.Automanage/preview/2021-04-30-preview/automanage.json

```diff
@@ -1611,6 +1657,17 @@
           "description": "Error message, if any, returned when deploying the resource."
         }
       }
+    },
+    "ServicePrincipalId": {
+      "description": "The Service Principal Id for the subscription.",
+      "type": "object",
+      "properties": {
+        "servicePrincipalId": {
```

Comment:
>servicePrincipalId [](http://example.com/codeflow?start=9&length=18)

Is this a secret? If so, it should be marked with the x-ms-secret extension. Ref: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret).

Reply:
This is not secret and this is just a GUID